spec/StringClassSpec.js

```diff
@@ -0,0 +1,202 @@
+require('../src/string-class');
+describe('test hasVowels method', () => {
+  it('should return true if the string has vowels', () => {
+    expect('TONIDA HAS ORANGES'.hasVowels()).toBeTruthy();
+    expect('TonidA'.hasVowels()).toBeTruthy();
+    expect('M37@i*, exp'.hasVowels()).toBeTruthy();
+    expect('mabishi'.hasVowels()).toBeTruthy();
+  });
+
+  it('should return false if the string has no vowels', () => {
+    expect('TBNRTY'.hasVowels()).toBeFalsy();
+    expect('tbnrty'.hasVowels()).toBeFalsy();
+    expect('245Gy'.hasVowels()).toBeFalsy();
+    expect('Thy'.hasVowels()).toBeFalsy();
+  });
```

Comment:
If you test for equality with built-in boolean values i.e `expect('Thy'.hasVowels()).toBe(false)` instead of  `expect('Thy'.hasVowels()).toBeFalsy()`, you can do away with the `should check the type of results to be boolean` test and make your tests shorter.